Backend/index.js

```diff
@@ -61,6 +101,39 @@ const verifyToken = (req, res, next) => {
   }
 };
 
+const updateImportantNotificationTypes = async () => {
```

Comment:
what is this function used for? it doesn't seem like importantTypes is being used anywhere?

Reply:
The function is called every 24 hours, updating the types of notifications that are important. it is called in the setInterval function